### ISSUE TITLE
Can't update 3.13 to RC1

### ISSUE THREAD

**Issue by edgarrmondragon** [glyph/MOPUp#351]
Using mopup installed with `pipx install mopup --python /usr/local/bin/python3.13 --suffix=@py313`, trying to upgrade Python 3.13 to RC1:

```console
$ mopup@py313 --interactive 1
Traceback (most recent call last):
  File "/Users/me/.local/bin/mopup@py313", line 8, in <module>
    sys.exit(main())
             ~~~~^^
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/mopup/__main__.py", line 32, in main
    libmain(interactive=interactive, force=force, minor_upgrade=minor, dry_run=dry_run)
    ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/mopup/__init__.py", line 40, in main
    thispkgver = Version(
        f"{thismajor}.{thisminor}.{thismicro}"
        + (f".{level}{serial}" if level != "final" else "")
    )
  File "/Users/me/.local/pipx/venvs/mopup@py313/lib/python3.13/site-packages/packaging/version.py", line 202, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '3.13.0.candidate1'
```

**Comment by glyph** [comment]
Hmm. Seems like this is a bug in https://github.com/pypa/packaging/ to me, but I am not sure if I'm reconstructing the version string properly here.

**Comment by glyph** [comment]
( Thanks for using MOPUp and thanks for the bug report :) )

**Comment by glyph** [comment]
Should be fixed in 2024.8.9, on PyPI now: https://pypi.org/project/MOPUp/2024.8.9/

**Comment by edgarrmondragon** [comment]
Awesome, thanks!
